README.md

```diff
@@ -63,7 +66,8 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
+Scope are the parent functions and means that an inner function will be influenced by it's parent. Children functions are not included in the scope and cannot be referenced. Calling variables and functions that are within the scope, but anything that is outside the scope cannot be referenced.
```

Comment:
I think this is partially correct. First scope does not just refer to functions, it includes variables and objects that you have access too as well.

The four rules can be summarized: 

```
1. Variables defined without the var keyword are always placed in the global scope.
2. Variables that are defined using the var keyword are created in the current local scope.
3. Every function and only functions create a new local scope.
4. The current scope includes all outer (enclosing) scopes.
```